### PR TITLE
Feanil/update course

### PR DESCRIPTION
**Pull request by feanil**
The deltas are really big because I imported the course into studio to make the updates and exported it back out.

fixes https://github.com/openedx/edx-demo-course/issues/32

### CODE REVIEW

File: html/700x_pathways.xml [hunk 1]
```diff
@@ -1 +1 @@
-<html filename="700x_pathways" display_name="Zooming Diagrams" graceperiod="18000 seconds" rerandomize="never" showanswer="attempted" start="2013-02-05T00:00:00Z" xqa_key="qaijS3UatK020Wc0sfCtFe0V6jpB4d64"/>
+<html filename="700x_pathways" display_name="Zooming Diagrams" graceperiod="18000 seconds" rerandomize="never" showanswer="attempted" start="2013-02-05T00:00:00+00:00" xqa_key="qaijS3UatK020Wc0sfCtFe0V6jpB4d64"/>
```
Comment (Carlos-Muniz) [inline html/700x_pathways.xml:1]: Does replacing `00Z` with `00+00:00` make any difference?

File: problem/700x_editmolB.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<problem display_name="Molecule Editor" graceperiod="0 day 5 hours 0 minutes 0 seconds" rerandomize="never" showanswer="attempted" start="2013-02-05T00:00" weight="" xqa_key="qaijS3UatK020Wc0sfCtFe0V6jpB4d64">
+<problem display_name="Molecule Editor" graceperiod="18000 seconds" rerandomize="never" showanswer="attempted" start="2013-02-05T00:00:00+00:00" weight="null" xqa_key="qaijS3UatK020Wc0sfCtFe0V6jpB4d64">
```
Comment (Carlos-Muniz) [inline problem/700x_editmolB.xml:1]: Why was this translated into seconds?

**Review by Carlos-Muniz — CHANGES_REQUESTED** [review]
Hey Feanil, looks pretty good. I'm pretty sure most of the small changes were done automatically via import/export, but just to make sure I have some questions.

Also, what did you do to test that this works correctly?

File: html/148ae8fa73ea460eb6f05505da0ba6e6.html [hunk 3]
```diff
@@ -3,6 +3,6 @@ <h2>Getting Your edX Certificate</h2>
 </div>
 <div>
 <p style="font-size: 16px;"><span style="line-height: 1.6;">If you passed your edX course, your certificate will be delivered online through&nbsp;</span><a style="line-height: 1.6;" href="http://www.edx.org/">edx.org</a><span style="line-height: 1.6;">. &nbsp;Be sure to check your email in the weeks following the final grading &ndash; you will be able to download and print your certificate.</span>&nbsp;Once the certificate generation process has completed, you will be able to download it from your profile page.</p>
-<p><img alt="" src="/c4x/edX/DemoX/asset/edX-SampleCertificate-Ux_x400.png" /></p>
+<p><img alt="" src="/static/edX-SampleCertificate-Ux_x400.png" /></p>
```
[the remark this answers is not in the document]
Comment (feanil) [reply html/148ae8fa73ea460eb6f05505da0ba6e6.html:6]: This was done by the import/export process, so I'm not sure exactly what's going on but I was able to import this course successfully afterwards so I think it's fine.

File: html/030e35c4756a4ddc8d40b95fbbfff4d4.html [hunk 4]
```diff
@@ -1,3 +1,4 @@
...
-<p style="margin-right: 0px; font-size: 16px; margin-left: 0px; font-family: 'Open Sans', Verdana, Geneva, sans-serif;">Watch the overview video (below), then click on "Example Week One" in the left hand navigation to get started.</p>
\ No newline at end of file
+<p><strong>Welcome to the Open edX Demo Course Introduction.</strong> This is where you can explore how to take an edX course (like this one). Most courses have an "intro" video that shows you how it all works. </p>
+<p style="margin-right: 0px; font-size: 16px; margin-left: 0px; font-family: 'Open Sans', Verdana, Geneva, sans-serif;">You can watch the introduction video (below) or scroll though the course studies and assignments using the toolbar (above).  Just for fun, we'll keep track of your work in this demo course, and show you your progress in the toolbar just like in a real course.</p>
+<p style="margin-right: 0px; font-size: 16px; margin-left: 0px; font-family: 'Open Sans', Verdana, Geneva, sans-serif;">Watch the overview video (below), then click on "Example Week One" in the left hand navigation to get started.</p>
+<style></style>
```
[the remark this answers is not in the document]
Comment (feanil) [reply html/030e35c4756a4ddc8d40b95fbbfff4d4.html:4]: Not sure why that was added, probably a quirk of the WYSIWYG editor.

File: html/700x_pathways.xml (hunk 1 above)
Comment (feanil) [reply html/700x_pathways.xml:1]: This was an automatic update, I think we're just now tracking the time in a slightly different format than we used to.

**Review by Carlos-Muniz — APPROVED** [review]
Thanks for commenting on each of my issues.
I approve!

**Pull request merged**